### DISCUSSION TITLE
Possible to filter by ambient light sensor?

### DISCUSSION THREAD

**Discussion by clevrdavid** [maca88/SmartBikeLights#26] [General]
This app is awesome and I have been able to set it up the way I want. 

I was just wondering if Garmin provide access to the ambient light sensor readings and whether this could be used for filtering, as it is potentially more accurate than setting filters by sunset/sunrise timespans.

**Comment by maca88** [comment]
Unfortunately Garmin does not provide an ambient light API, but there is a [feature request](https://forums.garmin.com/developer/connect-iq/i/bug-reports/ambient-light-sensor-api-access) on their forums which you can upvote, so that there will be a higher chance that it will be added in the future.

**Reply by clevrdavid** [reply]
Thanks, I have upvoted!

**Reply by maver1ck** [reply]
Thanks. I upvoted and asked about current status.

**Reply by cmeid** [reply]
Updooted
